servers/sunrpc/server.go

```diff
@@ -31,7 +42,7 @@ func New(l net.Listener) *SunRPC {
 	programsList = []Program{
 		newGfHandshake(),
 		newGfDump(),
-		newGfPortmap(),
+		pmap.NewGfPortmap(),
```

Comment:
@aravindavk This is what I was mentioning about. The code here doesn't care about where `pmap` package is located or if it's a glusterd2 plugin or not. If it satisfies the `Program` interface, it can hook into sunrpc server.